_posts/2024-02-10-unbunching.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Don't get your buses in a bunch
+author: 2TallTyler
+---
+
+Have you ever waited for a late bus, and then two arrived at once?
+
+This is called "[bus bunching](https://en.wikipedia.org/wiki/Bus_bunching)," and I bet you've seen it happen in OpenTTD -- and not just with buses.
+
+For years, players have requested an easier way to keep vehicles with shared orders spaced out along their route. In OpenTTD 14, we've added one.
+
+<!-- more -->
+
+![A comparison of red bunched buses versus purple unbunched buses]({% link /static/img/post_2024-02-06-bus-bunching/bunched-comparison.png %})
+
+To keep vehicles spaced evenly, we can either speed up or slow down vehicles.
+Speeding up vehicles is tricky even in real life, requiring adding extra vehicles, skipping stops, or turning a vehicle around before its final destination.
+It's much easier to simply slow them down. Real transit schedules have extra time built in to pad the schedule, often coinciding with the driver's breaks.
+A late-running vehicle can take a shorter break and depart on time. This is also possible in OpenTTD using timetables, but that's a lot of manual work.
+The most popular fork of OpenTTD, JGR's Patch Pack, includes an "autoseparation" feature that uses timetables automatically to keep vehicles spread out.
```

Comment:
that uses timetables automatically to -> that uses timetables to automatically
seems to read better?